src/Tizen.Network.Stc/Tizen.Network.Stc/StcEnumerations.cs

```diff
@@ -0,0 +1,323 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using Tizen.Internals.Errors;
+
+namespace Tizen.Network.Stc
+{
+    /// <summary>
+    /// Internal Enumeration for Stc time period.
+    /// </summary>
+    internal enum NativeTimePeriodType
+    {
+        /// <summary>
+        /// Unknown
+        /// </summary>
+        Unknown = 0,
+        /// <summary>
+        /// Hour
+        /// </summary>
+        Hour = 3600,
+        /// <summary>
+        /// Day
+        /// </summary>
+        Day = 86400,
+        /// <summary>
+        /// Week
+        /// </summary>
+        Week = 604800,
+        /// <summary>
+        /// Month
+        /// </summary>
+        Month = 2419200
+    }
+
+    /// <summary>
+    /// Enumeration for Stc time period.
+    /// </summary>
+    public enum TimePeriodType
+    {
+        /// <summary>
+        /// Hour
+        /// </summary>
+        Hour = 3600,
+        /// <summary>
+        /// Day
+        /// </summary>
+        Day = 86400,
+        /// <summary>
+        /// Week
+        /// </summary>
+        Week = 604800,
+        /// <summary>
+        /// Month
+        /// </summary>
+        Month = 2419200
+    }
+
+    /// <summary>
+    /// Internal Enumeration for network interface type.
+    /// </summary>
+    internal enum NativeNetworkInterface
+    {
+        /// <summary>
+        /// Network interface type is Unknown.
+        /// </summary>
+        Unknown = 0,
+        /// <summary>
+        /// Network interface type is datacall.
+        /// </summary>
+        Datacall,
+        /// <summary>
+        /// Network interface type is  wifi.
+        /// </summary>
+        Wifi,
+        /// <summary>
+        /// Network interface type is  wired.
+        /// </summary>
+        Wired,
+        /// <summary>
+        /// Network interface type is bluetooth.
+        /// </summary>
+        Bluetooth,
+        /// <summary>
+        /// Network interface type is all.
+        /// </summary>
+        All
+    }
+
+    /// <summary>
+    /// Enumeration for network interface type.
+    /// </summary>
+    public enum NetworkInterface
+    {
+        /// <summary>
+        /// Network interface type is datacall. 
+        /// </summary>
+        Datacall,
+        /// <summary>
+        /// Network interface type is  wifi.
+        /// </summary>
+        Wifi,
+        /// <summary>
+        /// Network interface type is  wired.
+        /// </summary>
+        Wired,
+        /// <summary>
+        /// Network interface type is bluetooth.
+        /// </summary>
+        Bluetooth,
+        /// <summary>
+        /// Network interface type is all.
```

Comment:
Description is wrong

Type for any network interface

Reply:
Done.